src/main/java/com/andersen/Ticket.java

```diff
@@ -0,0 +1,155 @@
+package com.andersen;
+
+import java.math.BigDecimal;
+
+public class Ticket implements Identifiable{
+
+	private static final int DISCOUNT = 2;
+
+	private int id;
+
+	@NullableWarning
+	private String concertHall;
+
+	@NullableWarning
+	private int eventCode;
+
+	private long timeStamp;
+
+	private boolean isPromo;
+
+	@NullableWarning
+	private char sector;
+
+	private double baggageLimit;
+
+	@NullableWarning
+	private BigDecimal price = BigDecimal.valueOf(20.50);
+
+	public Ticket() {
+		CheckNullFields.checkNullFields(this);
+	}
+
+	public Ticket(String concertHall, int eventCode, long timeStamp) {
```

Comment:
CheckNullFields.checkNullFields(this); 
mayve we need also add this here? what do you think?